packages/shared/src/lib/image.ts

```diff
@@ -256,6 +256,9 @@ export const cloudinaryIntegrationsSlackHeaderLight =
 export const cloudinaryIntegrationsSlackHeaderDark =
   'https://res.cloudinary.com/daily-now/image/upload/s--zzudPNtR--/f_auto,q_auto/v1721634746/public/Connection%20image%20-%20Dark';
 
+export const cloudinaryTopReaderBadgeBackground =
+  'https://daily-now-res.cloudinary.com/image/upload/s--hQ68GsbF--/f_auto,q_auto/v1730356819/webapp/image_7';
```

Comment:
Image is ~800x800 but badge is way more smaller. 
Do you think we could use a smaller one with repeat? 👀

Reply:
I replaced it with SVG version, so ~50% reduction in size.

Reply:
Nice!